src/test/index.test.ts

```diff
@@ -10,6 +10,8 @@ import { Genders } from './enums/genders';
 import { Role } from './enums/role';
 import { initDatabase, closeDatabase } from './utils/mongoConnect';
 import { getClassForDocument } from '../utils';
+import {Decimal128} from 'bson';
+import {fail} from 'assert';
```

Comment:
Just a cosmetic issue, but please add spaces after the opening and before the closing bracket.